install.sh

```diff
@@ -262,6 +262,14 @@ setup_binary() {
     info "Installing k3s to ${BIN_DIR}/k3s"
     $SUDO chown root:root ${TMP_BIN}
     $SUDO mv -f ${TMP_BIN} ${BIN_DIR}/k3s
+
+    if [ -x `command -v getenforce` ] && [ "Disabled" != `getenforce` ]; then
+        info "SeLinux is enabled, setting permissions"
+        if ! $SUDO semanage fcontext -l | grep "${BIN_DIR}/k3s" > /dev/null 2>&1; then
+            $SUDO semanage fcontext -a -t bin_t "${BIN_DIR}/k3s"
+        fi
+        $SUDO restorecon -v ${BIN_DIR}/k3s > /dev/null
```

Comment:
It has been awhile since I have used selinux, so I am not familiar with what exactly is happening here.

Will this work regardless of the selinux policy being used?

Is this permission too permissive for some administrators? Would it be better to provide guidance by pointing to Redhat docs on how to troubleshoot selinux permission issues?

Reply:
`semanage` manage creates a file context mapping to mark `k3s` as something that should be runnable. `restorecon` then takes that mapping and applies it to the file itself.

This will set the context if selinux is installed and in a `Permissive` or `Enforcing` policy (actually, anything other than `Disabled`).

I'm sure some admins might want more control, but for the average installer (i.e. someone that is actually using this install script) this change allows instant gratification even when selinux is being used.